docs/community/session-managers/agentcore-memory.md

```diff
@@ -10,29 +10,59 @@ The [AgentCore Memory Session Manager](https://github.com/aws/bedrock-agentcore-
 pip install 'bedrock-agentcore[strands-agents]'
 ```
 
+Or for local development:
+
+```bash
+git clone https://github.com/aws/bedrock-agentcore-sdk-python.git
+cd bedrock-agentcore-sdk-python
+uv sync
+source .venv/bin/activate
+```
+
```

Comment:
Given that we don't do this for all extensions/packages, i don't think we should encourage it for Bedrock specifically

```suggestion

```